custom_components/plugwise/binary_sensor.py

```diff
@@ -172,17 +167,15 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
         if self.entity_description.key != "plugwise_notification":
             return None
 
-        attrs: dict[str, list[str]] = {}
+        attrs: dict[str, list[str]] = {f"{severity}_msg": [] for severity in SEVERITIES}
```

Comment:
I'm not sure about these changes. I remember that I, on purpose, did not implement what's in Core due to the extra `notify_id` implemented in PW-beta.
Will look at this once more...

Reply:
Reverted & retagged accordingly

Reply:
note, this also affects L184-185

Reply:
Note-2-selves: this is to discard severities with empty lists - i.e. only showing severities with content instead of listing all severities including those without content.

Reply:
Added pre-line comments